src/openscad.cc

```diff
@@ -783,12 +783,42 @@ int gui(const vector<string> &inputFiles, const fs::path &original_path, int arg
 }
 #endif // OPENSCAD_QTGUI
 
+#include <execinfo.h>
```

Comment:
Yeah, that wasn't supposed to get committed as I can actually use a debugger now. I'll have an update shortly.